docs/importing-js-into-reason.md

```diff
@@ -22,3 +22,42 @@ module Greeting = {
 
 let make = () => <div> <Greeting name="Peter" /> </div>;
 ```
+
+### Default props and JavaScript components
+
+Sometimes you need your JavaScript component to have a default prop. How you
+handle this will depend on the details of the situation.
+
+#### Scenario 1: The JavaScript component prop is already coded to have a default
```

Comment:
do you mind adding a javascript example of what you mean here, ie:

```
function Greeting({ name = "Peter" }) {}
```

vs.

```
function Greeting({ name }) {}

Greeting.defaultProps = { name: "Peter" }
```

I know you probably mean the 2nd one but I think its worth being explicit, perhaps worth mentioning even version #1 (and what happens there)

Reply:
It's been a while since I've used ReactJS, but wouldn't either JS code result in the same Reason binding?  I'm fine adding the JS example in either style (or both), but let me know if I'm missing something.